maintainers/maintainer-list.nix

```diff
@@ -1492,6 +1492,12 @@
     githubId = 49904992;
     name = "Dawid Sowa";
   };
+  dbalan = {
```

Comment:
Please do you mind splitting this change into its separate commit?

Reply:
@mmahut Thanks for taking a look, I've split the change into two commits.